core/server/common/src/main/java/alluxio/master/journal/JournalSystem.java

```diff
@@ -145,6 +148,26 @@
    */
   boolean isFormatted() throws IOException;
 
+  /**
+   * @param master the master for which to add the journal sink
+   * @param journalSink the journal sink to add
+   */
+  default void addJournalSink(Master master, JournalSink journalSink) {}
+
+  /**
+   * @param master the master from which to remove the journal sink
+   * @param journalSink the journal sink to remove
+   */
+  default void removeJournalSink(Master master, JournalSink journalSink) {}
+
+  /**
+   * @param master the master from which to remove the journal sink
+   * @return a set of {@link JournalSink} for the given master, or all sinks if master is null
+   */
+  default Set<JournalSink> getJournalSinks(Master master) {
```

Comment:
are these default implementations necessary? I think it's safer to leave them non-default so that `JournalSystems` are required to specify how they manage journal sinks

Reply:
Done